unturned/OpenMod.Unturned/Building/Events/IUnturnedBarricadeEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OpenMod.Extensions.Games.Abstractions.Building;
+
+namespace OpenMod.Unturned.Building.Events
+{
+    public interface IUnturnedBarricadeEvent : IBuildableEvent
```

Comment:
Do we really need interfaces for Unturned events?

Reply:
Interfaces for both barricade and structure events were added as C# doesn't currently support multiple inheritances. These interfaces were added for two reasons:
- To establish a standard for the properties of UnturnedBarricadeBuildable and UnturnedStructureBuildable
- Allow for subscriptions for all updates based on buildable type (barricade and structures)